internal/ignition/ignition.go

```diff
@@ -1530,6 +1554,12 @@ func (g *installerGenerator) writeSingleHostFile(host *models.Host, baseFile str
 			return errors.Wrapf(errP, "Failed to parse machine cidr for node ip hint content")
 		}
 		setFileInIgnition(config, nodeIpHintFile, fmt.Sprintf("data:,KUBELET_NODEIP_HINT=%s", ip), false, 420, true)
+	} else if g.nodeIpAllocations != nil {
```

Comment:
can we have a function in common that will return true in case it is none platform multinode installation?